backend.go

```diff
@@ -33,11 +33,14 @@ func Factory(ctx context.Context, conf *logical.BackendConfig) (logical.Backend,
 		return nil, err
 	}
 
-	return b, nil
-
 	// TODO: is this run every time Vault starts up, or only when the secret engine is first mounted?
 	// if it's the latter, we'll need to find some way to instantiate the oauth2 client
 	// when Vault restarts
+	//
+	// For now, I'm only testing in an in-mem dev Vault, but this should be answered by running a persistent Vault
+	b.Logger().Info("New Plugin Backend Initialised")
```

Comment:
nit: I don't think we actually care about this anymore, now we've got a PeriodicFunc